src/vga.js

```diff
@@ -252,11 +344,17 @@ function VGAScreen(cpu, bus, vga_memory_size)
 
     this.diff_addr_min = this.vga_memory_size;
     this.diff_addr_max = 0;
+    this.diff_plot_min = this.vga_memory_size;
+    this.diff_plot_max = 0;
 
     this.dest_buffer = undefined;
 
     bus.register("screen-tell-buffer", function(data)
     {
+        if(this.dest_buffer && data[0])
+        {
+            data[0].set(this.dest_buffer.subarray(0,data[0].length));
+        }
```

Comment:
This copies the raw pixels from the old to the new buffer after a resize. Won't the result look distorted?

(minor: Missing space after `,`)

Reply:
> Won't the result look distorted?

For a brief moment and for most situations, probably yes. It's there just in case some programs decide to draw into the video memory before configuring the resolution, or decide to reuse the graphics already stored in the video memory, which I think should work fine with real VGA hardware.

Reply:
Sounds good.